Wikipedia/Code/WMFFeedContentSource.m

```diff
@@ -202,6 +202,23 @@ - (void)removeAllContentInManagedObjectContext:(NSManagedObjectContext *)moc {
 
 - (void)saveContentForFeedDay:(WMFFeedDayResponse *)feedDay pageViews:(NSDictionary<NSURL *, NSDictionary<NSDate *, NSNumber *> *> *)pageViews onDate:(NSDate *)date inManagedObjectContext:(NSManagedObjectContext *)moc completion:(dispatch_block_t)completion {
     [moc performBlock:^{
+        
+        NSString *key = [WMFFeedDayResponse WMFFeedDayResponseMaxAgeKey];
+        NSFetchRequest *request = [WMFKeyValue fetchRequest];
+        request.predicate = [NSPredicate predicateWithFormat:@"key == %@", key];
+        request.fetchLimit = 1;
+        NSArray<WMFKeyValue *> *results = [moc executeFetchRequest:request error:nil];
+        WMFKeyValue *keyValue = results.firstObject;
+        
+        if (keyValue == nil) {
+            NSEntityDescription *entity = [NSEntityDescription entityForName:@"WMFKeyValue" inManagedObjectContext:moc];
+            keyValue = [[WMFKeyValue alloc] initWithEntity:entity insertIntoManagedObjectContext:moc];
+            keyValue.key = key;
+            NSNumber *value = @(feedDay.maxAge);
+            keyValue.value = value;
+            [moc save:nil];
```

Comment:
The save here is unnecessary, the moc will be saved after all the content is imported.